tests/pytests/integration/modules/state/test_state_state_events.py

```diff
@@ -0,0 +1,107 @@
+"""
+tests.pytests.integration.modules.state.test_state_state_events
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+
+import logging
+import time
+
+import pytest
+
+log = logging.getLogger(__name__)
+
+
+EVENT_TAG = "salt/job/*/prog/{}/0"
```

Comment:
**chore (blocking):** globals are discouraged in pytests. please convert to a fixture.

Reply:
@barneysowood Just a heads up about the comment from @nicholasmhughes

Reply:
@nicholasmhughes replaced that with a fixture - let me know if that looks OK.